defaults/main.yml

```diff
@@ -35,41 +35,51 @@ osbs_buildroot_imagestream_live_tag: ''
 osbs_reactor_config_maps: []
 
 osbs_cronjob_api_version: batch/v2alpha1
-
-osbs_pruner_image: openshift3/ose
-osbs_pruner_command_build:
-- /usr/bin/oc
-- adm
-- prune
-- "--namespace={{ osbs_namespace }}"
-- builds
-- --orphans=true
-- --keep-complete=3
-- --confirm
-osbs_pruner_old_builds_command_build:
-- /usr/bin/oc
-- adm
-- prune
-- "--namespace={{ osbs_namespace }}"
-- builds
-- --orphans=true
-- --keep-younger-than=720h
-- --keep-complete=0
-- --keep-failed=0
-- --confirm
+osbs_pruner_builds_keep_complete: 1
+osbs_pruner_builds_keep_younger: '72h'
```

Comment:
@rcerven was this value change intentional? 720h against 72h

Reply:
yes

Reply:
thanks, looked like possible typo and I wanted to be sure. See that second MR uses same constant. Looks good to me